providers/base/units/input/test-plan.pxu

```diff
@@ -14,8 +14,9 @@ _name: Input tests (Manual)
 _description:
  Input tests (Manual)
 include:
- input/accelerometer                        certification-status=non-blocker
```

Comment:
Why was this job removed from the test plan?

Reply:
deleted it by accident, I've added it back.